src/api/artifact/blob/cache.go

```diff
@@ -0,0 +1,18 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package blob
+
+// TODO add cache
```

Comment:
If we enable cache in registry, there's already a cache of manifest in redis.

Reply:
It's the cache for both manifest and config layer

Reply:
Is it expensive for sending http request directly to get the data which is not large? If it is not, suggest removing cache out of the table so far. If one day, there is some perf issue appeared, then we can consider it at that time.

Reply:
As there is no OCI registry driver interface yet, we can keep the blob fetcher for now, and remove it util we define the driver interface